src/tool/subcommands/benchmark_cmd.rs

```diff
@@ -112,6 +128,36 @@ async fn benchmark_forest_encoding(
     Ok(())
 }
 
+// Exporting combines a graph traversal with ForestCAR.zst encoding. Ideally, it
+// should be no lower than `min(benchmark_graph_traversal,
+// benchmark_forest_encoding)`.
+async fn benchmark_exporting(
```

Comment:
non-blocking: could be useful to have this bench support export without compression, to be able to detect bottlenecks easier.
In future it would also be nice to have benchmarks for the actual code we use, e.g. https://github.com/ChainSafe/forest/blob/25be5be685edbe49d92dd46ccbd7044dc1d64e23/src/chain/mod.rs#L16

Then we'd be able to compare the barebones benchmark to the full version and see if we have any significant deviations.

Reply:
Isn't exporting without encoding a ForestCAR.zst file the same as a graph traversal?

Reply:
yes, you are right.